### DISCUSSION TITLE
"My Feed" replaces people by corporations 😡

### DISCUSSION THREAD

**Discussion by xlab** [community/community#54491] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, I've been using GitHub since 2010, and this is the place where I work on the code and discover code made by others. I've been following other people and their repos to be in the loop with what my friends, colleagues, and cool hackers alike work on.

Recent changes in GitHub basically removes the most important feed on the news I had every time I get there. It replaced people I know and repos I care about with some generic bullshit from large corporations. The picking makes no sense, the dev landscape is too variative and generic to be comprehended by any sort of recommendation engine.

Don't try to replace my news feed with an ad space for large corps and their incredibly starred repos. I don't care about them.

**Comment by licaon-kter** [comment]
Duplicate of https://github.com/orgs/community/discussions/13154 ? Use tab named "Following" instead of "For you" :shrug:

**Reply by nyghtly-derek** [reply]
That not very helpful unless users have a way to set "Following" as their default view. When you visit `github.com`, the first thing you see is "For You".
